nixos/modules/services/x11/picom.nix

```diff
@@ -301,13 +310,13 @@ in {
       };
 
       serviceConfig = {
-        ExecStart = "${pkgs.picom}/bin/picom --config ${configFile}";
+        ExecStart = "${cfg.package}/bin/picom --config ${configFile}";
```

Comment:
```suggestion
        ExecStart = "${getExe cfg.package} --config ${configFile}";
```

Reply:
picom doesn't specify `meta.mainProgram`, so this won't work, right?

Reply:
Oh, I did not notice than.  However, it works, but will emit a warning about not setting `matemainProgram`.

Reply:
https://github.com/NixOS/nixpkgs/commit/2ed62c15fb34c25f578c5078656046e3f42de2b5